app/src/main/java/org/covidwatch/android/CurrentUserExposureNotifier.kt

```diff
@@ -1,74 +0,0 @@
-package org.covidwatch.android
-
-import android.app.Application
-import android.app.NotificationChannel
-import android.app.NotificationManager
-import android.content.Context
-import android.os.Build
-import androidx.core.app.NotificationCompat
-import androidx.core.app.NotificationManagerCompat
-import androidx.lifecycle.Observer
-import org.covidwatch.android.data.CovidWatchDatabase
-import org.covidwatch.android.data.contactevent.LocalContactEventsViewModel
-
-class CurrentUserExposureNotifier(var application: Application) {
-
-    private val viewModel: LocalContactEventsViewModel =
-        LocalContactEventsViewModel(
-            CovidWatchDatabase.getInstance(application).contactEventDAO(),
-            application
-        )
-
-    fun startObservingLocalContactEvents() {
-
-        createNotificationChannel()
-
-        viewModel.contactEvents.observeForever(Observer {
```

Comment:
Deleted this implementation (file) because here we might have a memory leak because we are not removing observer.